easydock/protonation.py

```diff
@@ -86,6 +86,9 @@ def read_smiles(fname):
 def protonate_pkasolver(input_fname: str, output_fname: str, ncpu: int = 1):
     import torch
     from pkasolver.query import QueryModel
+
+    estimated_smi = os.path.getsize(input_fname)//70
+    chunksize = min(max(0, estimated_smi / ncpu), 500)
```

Comment:
Replace please `0` with `1` to set the range from 1 to 500. That was my mistake.